pyexchange/erisx.py

```diff
@@ -81,16 +81,25 @@ def __init__(self, fix_trading_endpoint: str, fix_trading_user: str,
         assert(isinstance(clearing_url, str) or (clearing_url is None))
         assert(isinstance(api_key, str) or (api_key is None))
         assert(isinstance(api_secret, str) or (api_secret is None))
+        assert(isinstance(certs, dict) or (certs is None))
         assert(isinstance(account_id, int))
 
         # enable access from sync_trades and inventory_service without overriding socket
         if fix_trading_endpoint is not None and fix_trading_user is not None:
-            self.fix_trading = ErisxFix(fix_trading_endpoint, fix_trading_user, fix_trading_user, password)
+            if certs is None:
```

Comment:
The switch seems to add an extra 6 lines of code.  You could just pass `certs` all the time, even if it's `None`.

Reply:
Good point, updated accordingly.